Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/JobViewManager.java

```diff
@@ -5,15 +5,12 @@
 
 package com.microsoft.azure.hdinsight.common;
 
+import com.microsoft.azure.hdinsight.sdk.cluster.IClusterDetail;
 import com.microsoft.azure.hdinsight.sdk.rest.spark.Application;
 import com.microsoft.azure.hdinsight.spark.jobs.ApplicationKey;
-import com.microsoft.azure.hdinsight.spark.jobs.framework.JobViewPanel;
-import com.microsoft.azure.hdinsight.sdk.cluster.IClusterDetail;
 import com.microsoft.azuretools.azurecommons.helpers.NotNull;
 import com.microsoft.azuretools.azurecommons.helpers.Nullable;
-import javafx.util.Pair;
 
-import java.util.HashMap;
```

Comment:
Looks we can ignore these changes.

Reply:
JobViewPanel is imported but not used in this file. If we do no change to this file, ClassNotFoundException will be reported when compiling this file.